### PR TITLE
add stats listener to knative and ingress proxy configmaps in helm chart

### PR DESCRIPTION
**Pull request by ilackarms**
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/821

### CODE REVIEW

File: install/helm/gloo/templates/12-ingress-proxy-configmap.yaml [hunk 1]
```diff
@@ -34,6 +34,40 @@ data:
...
+                            - "*"
+                          routes:
+                            - match:
+                                prefix: "/metrics"
+                                headers:
+                                - name: ":method"
+                                  exact_match: GET
+                              route:
+                                prefix_rewrite: "/stats/prometheus"
+                                cluster: admin_port_cluster
```
Comment (yuval-k) [inline install/helm/gloo/templates/12-ingress-proxy-configmap.yaml]: you did not add the cluster - this config will not work

File: install/helm/gloo/templates/12-ingress-proxy-configmap.yaml [hunk 2]
```diff
@@ -34,6 +34,55 @@ data:
...
+          cluster_name: admin_port_cluster
+          endpoints:
+          - lb_endpoints:
+            - endpoint:
+                address:
+                  socket_address:
+                    address: 127.0.0.1
+                    port_value: 19000
+{{- end}} # if Values.ingressProxy.deployment.stats
+
```
Comment (rickducott) [inline install/helm/gloo/templates/12-ingress-proxy-configmap.yaml]: There are 2 if blocks on the same condition in a row

File: install/helm/gloo/templates/15-clusteringress-proxy-configmap.yaml [hunk 3]
```diff
@@ -33,6 +33,56 @@ data:
...
+          cluster_name: admin_port_cluster
+          endpoints:
+          - lb_endpoints:
+            - endpoint:
+                address:
+                  socket_address:
+                    address: 127.0.0.1
+                    port_value: 19000
+{{- end}} # if Values.settings.integrations.knative.proxy.stats
+
```
Comment (rickducott) [inline install/helm/gloo/templates/15-clusteringress-proxy-configmap.yaml]: Same as above

**Pull request merged**